esp_pytest/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.7-buster
+
+RUN : \
+    && export PIP_EXTRA_INDEX_URL="https://www.piwheels.org/simple" \
+    && pip install \
+        pytest-embedded \
+        pytest-embedded-idf \
+        pytest-embedded-serial-esp \
```

Comment:
How will we update this image?

Can we fix versions of pytest here and update it manually when needed?
Or will we automatically update the image when new version of pytest is released?